dev/CommonStyles/TextBox_themeresources.xaml

```diff
@@ -25,11 +25,12 @@
             <SolidColorBrush x:Key="TextBoxDisabledBorderThemeBrush" Color="#66FFFFFF" />
             <SolidColorBrush x:Key="TextBoxDisabledForegroundThemeBrush" Color="#FF666666" />
             <SolidColorBrush x:Key="TextBoxForegroundThemeBrush" Color="#FF000000" />
-            <StaticResource x:Key="TextControlBackgroundFocused" ResourceKey="SystemAltHighColor" />
+            <StaticResource x:Key="TextControlBackgroundFocused" ResourceKey="SystemControlBackgroundAltHighBrush" />
             <StaticResource x:Key="TextControlBorderBrush" ResourceKey="SystemControlForegroundBaseMediumBrush" />
             <StaticResource x:Key="TextControlBorderBrushPointerOver" ResourceKey="SystemControlHighlightBaseMediumHighBrush" />
-            <StaticResource x:Key="TextControlButtonForeground" ResourceKey="SystemBaseHighColor" />
-            <StaticResource x:Key="TextControlForegroundFocused" ResourceKey="SystemBaseHighColor" />
+            <StaticResource x:Key="TextControlButtonForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
```

Comment:
Originally `TextControlButtonForeground` references `SystemChromeBlackMediumColor`, which has the same value as `SystemBaseMediumHighColor` (in light theme), so it might make sense to use `SystemControlForegroundBaseMediumHighBrush` here.

Reply:
Great catch! Switched those out now.